src/main/pages/che-7/end-user-guide/ref_che-theia-plug-in-metadata.adoc

```diff
@@ -91,20 +91,20 @@ The link:https://github.com/eclipse/che-plugin-registry/tree/master/v3/plugins[c
 :===
 
 
-.Example `meta.yaml` for a Che-Theia plug-in: the Che machine-exec Service
+.Example `meta.yaml` for a Che-Theia plug-in: the {prod-short} machine-exec Service
 [source,yaml]
 ----
 apiVersion: v2
 category: Other
-description: Che Plug-in with che-machine-exec service to provide creation terminal or tasks for Eclipse CHE workspace containers.
-displayName: Che machine-exec Service
+description: {prod-short} Plug-in with che-machine-exec service to provide creation terminal or tasks for Eclipse CHE workspace containers.
+displayName: {prod-short} machine-exec Service
 firstPublicationDate: "2019-12-02"
 icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
 name: che-machine-exec-plugin
 publisher: eclipse
 repository: https://github.com/eclipse/che-machine-exec/
 title: Che machine-exec Service Plugin
-type: Che Plugin
+type: {prod-short} Plugin
```

Comment:
```suggestion
type: {prod-short} plug-in
```

Reply:
Hmmm... looking at this again... is `Che Plugin` a defined 'type'? If it is, then neither an attribute for prod name, nor the 'plug-in' form should be used.

Reply:
```suggestion
type: Che Plugin
```